src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/ArtworkItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Text, Clickable, Column, Flex, SkeletonBox, Box } from "@artsy/palette"
+import { SelectArtworkItem } from "./SelectArtworkItem"
+
+interface ArtworkItemProps {
+  item: any
+  onItemPress: (item: any) => void
+}
+
+export const ArtworkItem: React.FC<ArtworkItemProps> = ({
+  item,
+  onItemPress,
+}) => {
+  const [isSelected, setIsSelected] = React.useState(false)
+
+  const onClick = () => {
+    onItemPress(item)
+    setIsSelected(!isSelected)
+  }
+
+  return (
+    <Column span={[6, 4]}>
```

Comment:
**suggestion**: move `Column` to the `ArtworksList` component. Something like this

```javascript
<GridColumns>
  {TEMP_ARTWORKS.map(artwork => {
    return (
      <Column span={[6, 4]}>
        <ArtworkItem item={artwork} onItemPress={handleItemPress} />
      </Column>
    )
  })}
</GridColumns>
```